diesel/src/migration/mod.rs

```diff
@@ -19,3 +20,67 @@ pub trait Migration {
         None
     }
 }
+
+/// A migration name
```

Comment:
Can we document what this actually does rather than just repeating the name of the type?